airflow/bin/cli.py

```diff
@@ -72,15 +72,30 @@ def sigquit_handler(sig, frame):
     print("\n".join(code))
 
 
-def setup_logging(filename):
-    root = logging.getLogger()
+def setup_file_logging(filename, fmt=settings.SIMPLE_LOG_FORMAT):
     handler = logging.FileHandler(filename)
```

Comment:
@bolkedebruin I've taken over this issue and am planning to implement more configuration as you suggest. I was wondering if it might be cleaner in these circumstances to merge @mtdewulf's code and I will then work on top of it?